common/cf-guest/build.rs

```diff
@@ -0,0 +1,7 @@
+fn main() -> std::io::Result<()> {
+    prost_build::Config::new()
+        .enable_type_names()
+        .type_name_domain(["."], "composable.finance")
```

Comment:
can we change to Picasso?

Reply:
The idea is that the domain corresponds to an actual web domain. Though I guess sit wouldn’t be end of the world if we changed it to picasso.